job-descriptions/software-engineer.md

```diff
@@ -40,15 +40,17 @@ Take a look at our public [near-term product roadmap](https://docs.sourcegraph.c
 
 ### Interview process
 
-1.  You [apply here](https://hire.withgoogle.com/public/jobs/sourcegraphcom/view/P_AAAAAADAAADP_pY7jAAAXU).
-2.  We schedule a 30 minute introductory phone call to tell you more about Sourcegraph, to share what we are looking for in a candidate, and to find out what you are looking for in your next role.
-3.  We collect a code sample from one or more of the following sources:
-    - We review any existing public code that you have (e.g. your own projects, or your contributions to other projects).
-    - You work on a paid project with us.
-    - We give you a 2 hour coding assignment to complete.
-4.  We schedule a few more hours of technical and non-technical interviews. We are happy to fly you out to our San Francisco office, or conduct the remaining interviews over video chat, whatever works best for you.
-5.  We check your references.
-6.  We make you a job offer.
+1.  You [apply here](https://hire.withgoogle.com/public/jobs/sourcegraphcom/view/P_AAAAAADAAADP_pY7jAAAXU)!
+1.  We schedule a 30 minute call to chat with you about Sourcegraph and to find out what you are looking for in your next role.
+1.  Choose your adventure!
+    - **Coding assignment**: We give you a well defined problem to solve and a 2 hour time limit. This is a lower time commitment than the paid project.
+    - **Paid project**: You propose working on one of the open issues in our open source repositories. Once we approve your proposal, you work on that issue until it is finished and merged. This is a higher time commitment than the coding assignment but it gives you more control to showcase your abilities and we will pay you a flat rate depending on the size of the project.

```

Comment:
Can we add the actual $ range? E.g. "flat rate ($X–$Y)"?

And this sort of sounds like the people who can invest more time to do the paid project are going to be better off/have a better shot in the interview process. Is that actually true? Do we want it to be true?

Reply:
Yeah, will add the range.

> And this sort of sounds like the people who can invest more time to do the paid project are going to be better off/have a better shot in the interview process. Is that actually true? Do we want it to be true?

No this is not true and I don't want it to sound that way. Do you have any suggestions about how to tweak the wording here?

Reply:
Yeah, great. The language "gives you more control to showcase your abilities" is the phrase I'm thinking of. 

Perhaps just "gives you more control", since it does, in regards to things like choosing your own project in your own area of interest. 

Or perhaps "gives you more control and a more realistic view of what working at Sourcegraph is like"

Reply:
I just removed that part to keep in neutral.